### PR TITLE
Go to screen action implemented in application form reducer

### PR DESCRIPTION
**Pull request by arnifreyrs**
Improvements to Review screen.

Translations in parental leave form finished for now.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by sindrig — APPROVED** [review]
LGTM

**Review by sindrig — APPROVED** [review]

File: libs/application/templates/parental-leave/src/lib/dataSchema.ts [hunk 1]
```diff
@@ -51,13 +50,11 @@ export const dataSchema = z.object({
...
     name: z.string().nonempty(),
-    nationalRegistryId: z
-      .string()
-      .nonempty()
-      .refine(
-        (n) => kennitala.isValid(n) && kennitala.isCompany(n),
-        'Kennitala þarf að vera gild',
-      ),
+    nationalRegistryId: z.string().nonempty(),
+    // .refine(
```
Comment (madebynoam) [inline libs/application/templates/parental-leave/src/lib/dataSchema.ts:54]: Should we add a note for the commented out code?

**Review by madebynoam — APPROVED** [review]
Lgtm, just one small comment about the commented out code.

**Review by sindrig — APPROVED** [review]

File: libs/application/templates/parental-leave/src/lib/dataSchema.ts (hunk 1 above)
Comment (arnifreyrs) [reply libs/application/templates/parental-leave/src/lib/dataSchema.ts:54]: this will be removed in Bærings work

**Pull request merged**